src/views/product-landing/components/cards/index.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react'
 import InlineSvg from '@hashicorp/react-inline-svg'
 import classNames from 'classnames'
-import tagIconDict from './tag-icon-dict.json'
+import iconDict from './icon-dict'
+import tagIconDict from './tag-icon-dict'
 import { CardProps } from './types'
 import CardLink from 'components/card-link'
 import Text from 'components/text'
 import s from './style.module.css'
+import IconTile from 'components/icon-tile'
 
 function Cards({ columns, cards }: CardProps): React.ReactElement {
   return (
     <div className={s.root}>
       <div className={classNames(s.cards, s[`columns-${columns}`])}>
         {cards.map((card, idx) => {
-          const { iconSvg, heading, text, tags, url } = card
+          const { icon, iconBrandColor, heading, text, tags, url } = card
```

Comment:
JSON now uses `icon` keys, rather than full icon SVGs. While not ideal for bundle size, the compromise on that front seemed worth the much cleaner JSON. Definitely not suggesting this as a final solution - focus here is on getting placeholder content in front of product stakeholders, and setting up mechanisms that will allow us to iterate on the content at their pace.